static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-a1-a2-thrasher.js

```diff
@@ -54,7 +59,7 @@ define([
             if (this.thrasher()) {
                 var linkEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button');
                 if (linkEl && linkEl.getAttribute('href')) {
-                    linkEl.setAttribute('href', config.page.membershipUrl + '/bundles?INTCMP=MEMBERSHIP_AB_THRASHER_' + config.page.edition + '_' + variant.toUpperCase());
+                    linkEl.setAttribute('href', config.page.membershipUrl + '/bundles?INTCMP=MEMBERSHIP_A_THRASHER_' + config.page.edition + '_' + variant.toUpperCase());
```

Comment:
I suspect they bypassed the thrasher and edited the url directly, since we only saw one of those. But maybe to be double-certain a force to upper case wouldn't hurt.